langlib/lang.query/src/main/ballerina/helpers.bal

```diff
@@ -128,7 +141,15 @@ function toString(stream<Type, CompletionType> strm) returns string|error {
     return result;
 }
 
-function addToTable(stream<Type, CompletionType> strm, table<map<Type>> tbl, error? err) returns table<map<Type>>|error {
+function addToTable(stream<Type, CompletionType> strm, table<map<Type>> tbl, error? err, boolean isReadOnly) returns table<map<Type>>|error {
+    if isReadOnly {
+        table<map<Type>> tbl2 = table [];
```

Comment:
The created readonly keyed tables don't seem to work as expected
```ballerina
type Rec record {|
    readonly int id;
|};

public function main() returns error? {
    table<Rec> key(id) & readonly tbl = check table key(id) from var i in 1...3
                                      select {
                                          id: i
                                      };
    Rec? member1 = tbl[1]; // panics with `"cannot find key '1'"`

    io:println(member1);
}
``` 

Without readonly this works as expected

Reply:
will check.

Reply:
Fixed in https://github.com/ballerina-platform/ballerina-lang/pull/36576/commits/0f59daaa2d84f5dfb98bb3781dd01ab4f33f8e75.